content/en/installing-pixie/03-install-guides/01-community-cloud-for-pixie.md

```diff
@@ -120,7 +120,7 @@ For more information, check out our [Live UI guide](/using-pixie/using-live-ui/)
 
 ### Check out the tutorials
 
-Learn how to use Pixie for [Network Monitoring](/tutorials/pixie-101/network-monitoring/), [Infra Health](/tutorials/pixie-101/infra-health/), [Database Query Profiling](/tutorials/pixie-101/database-query-profiling/), and more.
+Learn how to use Pixie for [Network Monitoring](/tutorials/pixie-101/network-monitoring/), [Infra Health](/tutorials/pixie-101/infra-health/), [Service Health](/tutorials/pixie-101/service-health/), [Database Query Profiling](/tutorials/pixie-101/database-query-profiling/) or [Request Tracing](/tutorials/pixie-101/request-tracing/).
```

Comment:
From the deploy preview, it looks like there's a lot of text here. Should we make it a bulleted list instead so that it's easier to parse?